qiskit_dynamics/solvers/solver_classes.py

```diff
@@ -523,6 +524,8 @@ def solve(
             Array.default_backend() == "jax"
             and (method == "jax_odeint" or _is_diffrax_method(method))
             and all(isinstance(x, Schedule) for x in signals_list)
+            # check if jit transformation is already performed.
+            and not (isinstance(jnp.array(0), core.Tracer))
```

Comment:
This is related with the issue https://github.com/Qiskit/qiskit-dynamics/issues/175